### PR TITLE
Rename master version as v1.0.0

### PR DESCRIPTION
**Pull request by tiraboschi**

### CODE REVIEW

File: deploy/olm-catalog/kubevirt-hyperconverged/kubevirt-hyperconverged.package.yaml [hunk 1]
```diff
@@ -1,9 +1,11 @@
 packageName: kubevirt-hyperconverged
 channels:
+- name: "1.0.0"
+  currentCSV: kubevirt-hyperconverged-operator.v1.0.0
 - name: "0.0.3"
-  currentCSV: kubevirt-hyperconverged-operator.v0.0.3
+  currentCSV: kubevirt-hyperconverged-operator.v0.0.2
```
Comment (rthallisey) [inline deploy/olm-catalog/kubevirt-hyperconverged/kubevirt-hyperconverged.package.yaml]: this doesn't look right

File: templates/olm-catalog/kubevirt-hyperconverged/kubevirt-hyperconverged.package.yaml.in [hunk 2]
```diff
@@ -1,9 +1,9 @@
 packageName: kubevirt-hyperconverged
 channels:
-- name: "0.0.3"
-  currentCSV: kubevirt-hyperconverged-operator.v0.0.3
+- name: "1.0.0"
+  currentCSV: kubevirt-hyperconverged-operator.v1.0.0
```
Comment (rthallisey) [inline templates/olm-catalog/kubevirt-hyperconverged/kubevirt-hyperconverged.package.yaml.in]: Are you renaming 0.0.3 to 1.0.0? Or adding 1.0.0?

File: deploy/olm-catalog/kubevirt-hyperconverged/kubevirt-hyperconverged.package.yaml (hunk 1 above)
Comment (tiraboschi) [reply deploy/olm-catalog/kubevirt-hyperconverged/kubevirt-hyperconverged.package.yaml]: Done, thanks

File: templates/olm-catalog/kubevirt-hyperconverged/kubevirt-hyperconverged.package.yaml.in (hunk 2 above)
Comment (tiraboschi) [reply templates/olm-catalog/kubevirt-hyperconverged/kubevirt-hyperconverged.package.yaml.in]: Adding 1.0.0

**Comment by rthallisey** [comment]
So we can follow an x.y.z stream release plan in operatorhub.   Right now, we're basically doing z-stream releases which isn't accurate.

**Pull request merged**